### ISSUE TITLE
Contribute to GATK to improve ReadFilter documentation

### ISSUE THREAD

**Issue by magicDGS** [magicDGS/ReadTools#271]
After #270, we will have the generated documentation from the ReadFilters, but it is incomplete. We should scan our own documentation page and submit a PR to GATK to fix any problem.

**Comment by magicDGS** [comment]
Added PR https://github.com/broadinstitute/gatk/pull/3128

**Comment by magicDGS** [comment]
Already done. It will populate in our help for next release including the fix. Closing.
